DuckDuckGo/NetworkProtectionConvenienceInitialisers.swift

```diff
@@ -28,7 +28,7 @@ import Subscription
 
 private class DefaultTunnelSessionProvider: TunnelSessionProvider {
     func activeSession() async -> NETunnelProviderSession? {
-        try? await ConnectionSessionUtilities.activeSession()
+        return await AppDependencyProvider.shared.networkProtectionTunnelController.activeSession()
     }
 }
```

Comment:
We could remove `DefaultTunnelSessionProvider` entirely now, since it just delegates through to the tunnel controller. I'll push a commit to solve this.